product_docs/docs/migration_portal/4.0/04_mp_migrating_database/03_mp_schema_migration.mdx

```diff
@@ -78,77 +78,48 @@ The converted schemas migrate to the target server.
 
 You can install new EDB Postgres Advanced Server on premises on Windows or Linux platforms and migrate the schemas.
 
-### Migrate schemas on Windows
-
-Migrate schemas to a new on-premises EDB Postgres Advanced Server on Windows.
-
 1.  Select **New on-premises EDB Postgres Postgres Advanced Server**.
 
 
 2.  Select one or more schemas to migrate on EDB Postgres Advanced Server.
 
 
-3.  For the operating system, select **Windows**.
-
-
-4.  Select **Download Windows installer**.
+3.  For the operating system, select **Windows** or **Linux**.
 
+    - *On Windows*:   
+ 
+     Select **Download Windows installer**. For information on the installation procedure, select **EDB Postgres Advanced Server Installation Guide for Windows**.
 
-5.  For installation steps, select **EDB Postgres Advanced Server Installation Guide for Windows**.
+    - *On Linux*:
+ 
+     Select either **Repository** or **More options** to install the EDB Postgres Advanced Server. For information on the installation procedure, select **EDB Postgres Advanced Server Installation Guide** for Linux.
 
 
 6.  To download the assessed schemas, select **Download SQL file**.
 
 
 7.  You can import schemas by running:
 
-    -   On CLI
-
-        ```text
-        \i c:\users\...\<project_name>.sql
-        ```
-
-    -   On cmd/shell
-
-        ```text
-        edb-psql -f <project_name>.sql
-        ```
-
- !!! Note
-    You can alternatively use the `pgAdmin` client for the import.
-
-
-### Migrate schemas on Linux
-
-Migrate schemas to an on-premises EDB Postgres Advanced Server on Linux.
-
-1.  Select **New On-premises EDB Postgres Advanced Server**.
-
-
-2.  Select one or more schemas to migrate on EDB Postgres Advanced Server.
-
-3.  For the operating system, select **Linux**.
-
-
-4.  You can select one of the following options to install the EDB Postgres Advanced Server:
-
-    -   Repository
-    -   More options
-
-5.  For information on the installation procedure, select **EDB Postgres Advanced Server Installation Guide** for Linux.
-
-
-6.  To download the assessed schemas, select **Download SQL file**. 
+    - *On Windows CLI*:  
```

Comment:
I don't know what Windows CLI means here. Based on the next command it seems like it is referring to the psql, but not sure. If does refer to psql, then the label On Windows CLI should be updated to In psql client.

Reply:
Addressed as per last comment below.